pkgs/development/libraries/science/math/clblast/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, opencl-headers
+, ocl-icd
+, OpenCL
+}:
+
+stdenv.mkDerivation rec {
+  pname = "clblast";
+  version = "1.5.3";
+
+  src = fetchFromGitHub {
+    owner = "CNugteren";
+    repo = "CLBlast";
+    rev = "${version}";
```

```suggestion
    rev = version;
```
or `rev = "refs/tags/${version}";`